Sources/PinLayout.swift

```diff
@@ -123,7 +123,7 @@ public class PinLayout<View: Layoutable> {
         let layoutFrame = view.keyboardLayoutGuide.layoutFrame
         guard !layoutFrame.isEmpty else { return .zero }
         
-        return UIEdgeInsets(top: layoutFrame.origin.y,
+        return UIEdgeInsets(top: layoutFrame.height,
```

Comment:
Are you sure about this, the keyboard guide represents the area of the keyboard that is covering the view. In this screenshot its the red section for the yellow UIView.

![image](https://user-images.githubusercontent.com/14981341/151999114-8111a7fa-79af-4219-b01f-eaa364a46e40.png)

Also, sorry about this, but in reality `keyboardLayoutGuide` should maybe not be exposed as an `UIEdgeInsets`, but by a CGRect, i.e. directly by the `view.keyboardLayoutGuide.layoutFrame` 🤔🤔 Because it really represent a section/zone/area, and not margins as we are used to. So finally, maybe I would expose it as `var keyBoardRect CGRect`

What do you think?

Reply:
omg... I guess I'm out of my mind. Sorry.

Reply:
What do you think of renaming the property? I can do it if you prefer. Its good to do it before anyone use it.

Reply:
> @baekteun, what do you think of renaming the property? I can do it if you prefer. Its good to do it before anyone use it.

I'm sure you'll do a great job.